tests/arcjet/test-bot-protection.js

```
@@ -0,0 +1,219 @@
+const { config } = require('dotenv');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)